consensus_encoding/src/encode/encoders.rs

```diff
@@ -249,6 +311,22 @@ impl Encoder for CompactSizeEncoder {
     }
 }
 
+/// A type to hold the encoded value of a `CompactSize` integer value.
+pub struct CompactSizeSlice([u8; SIZE], usize);
```

Comment:
In 9f9b6d65254d9ae5d2d188bb03e87913111a83d5:

`CompactSizeSlice` should not be public